service/history/task/task.go

```diff
@@ -236,10 +236,12 @@ func (t *taskImpl) HandleErr(
 			t.attempt++
 			if t.attempt > t.criticalRetryCount() {
 				t.scope.RecordTimer(metrics.TaskAttemptTimerPerDomain, time.Duration(t.attempt))
-				t.logger.Error("Critical error processing task, retrying.",
+				t.logger.Error("Critical error processing task, retrying.", // TODO: there might be more. If there is, do the same thing
 					tag.Error(err),
 					tag.OperationCritical,
+					// fixme!!! Add a new tag attempt into TaskType.log the attempt
```

Comment:
It is better to create a new ticket for it and link to your current ticket or just fix it if it is a simple fix.

Reply:
Ohh I forgot to delete those comments. Those were reminders for myself. Thanks Neil.

Reply:
The comments were deleted.